Topics/Development_Process/App_Store_Deployment.md

```diff
@@ -0,0 +1,144 @@
+# Apple App Store Deployment
+
+## Introduction
+After working towards a useable version of your mobile app, your last step would be to handle the deployment side. 
+The main difficulties faced with submitting an app to the App Store is that they set high standards for privacy and security. 
+You would need various certificates in order to distribute your app to the App Store as well as use certain features like: Push Notifications, Apple Pay, etc.. 
+These are some possible hurdles to keep in mind when working on a project to release an iOS app, but they certainly help with raising the quality of your project.
+
+## Requirements
+- Apple Developer Account
+- Apple Certificates
+- Xcode (Requires a MacBook)
+- Established Privacy Policy & Terms and Conditions
+
+
+### Apple Developer Program
+The most crucial component you will need to be able to deploy to the App Store is to enroll in the Apple Developer Program. 
+You may enroll as an individual or as an organization, but keep in mind that they have different requirements for each option.
+
+
+**EXTRA REQUIREMENTS**:
+
+    Individual
+    - Verified Two Factor Authentication
+    - Basic Personal Information
+
+    Organization
+    - Verified Two Factor Authentication
+    - D-U-N-S Number
+    - Legal Entity Status
+    - Legal Binding Authority
+    - Organization Website
+
+The full details are available on the Apple Developer Program enrollment page: [Apple Developer Program](https://developer.apple.com/programs/enroll/)
+    
+
+### iOS-Related Certificates
+
+
+- **Apple Push Notification Service** Certificate (Notifications)
+- **Apple Pay Payment** Processing Certificate (Apple Pay)
+- **Apple Pay Merchant** Identity Certificate (Apple Pay)
+- **Pass Type** ID Certificate (Apple Wallet)
+- **iOS Distribution** Certificate (Auto-Generated on Account Creation)
+
```

Comment:
Consider going into more detail about what each one of them means. Most people who are just getting into it may not know this terminology.

Reply:
This looks good now.